appveyor.yml

```diff
@@ -13,7 +13,7 @@ environment:
 configuration:
   - Debug
   - Release
-version: 3.14.0.{build}
+version: 3.15.0.{build}
```

Comment:
Since you've extended the definition of the interface `IClassifier`, this is breaking change now. Please update to `4.0.0`.

Reply:
done